Source/Diffusion.cpp

```diff
@@ -73,14 +70,13 @@ PeleC::getMOLSrcTerm(
   const amrex::GpuArray<amrex::Real, AMREX_SPACEDIM> dxD = {
     {AMREX_D_DECL(dx1, dx1, dx1)}};
 
-  // Grab the BCs
+  // BCs
   const amrex::StateDescriptor* desc = state[State_Type].descriptor();
   const auto& bcs = desc->getBCs();
   amrex::Gpu::DeviceVector<amrex::BCRec> d_bcs(desc->nComp());
   amrex::Gpu::copy(
     amrex::Gpu::hostToDevice, bcs.begin(), bcs.end(), d_bcs.begin());
 
-  // Fetch some gpu arrays
   prefetchToDevice(S);
   prefetchToDevice(MOLSrcTerm);
```

Comment:
I never see these in other applications. Do we know if they even do anything?

Reply:
Yeah I have no idea. I think this is a relic. It also is only used for managed mem: https://github.com/AMReX-Codes/amrex/blob/96b811d2cb9f24e87428e4ebc9904b4dcdf73e7f/Src/Base/AMReX_BaseFab.H#L1689. 

So I will just remove these.